src/views/TheWalletView.vue

```diff
@@ -177,26 +177,25 @@ export default {
           });
         } catch (er) {
           const { message } = er;
-          let toastMsg = '';
+          let toastMsg = ' ';
           let toastLink = {};
           if (message) {
             if (message.includes('pending')) {
               toastMsg =
                 'There is a pending request to MetaMask, make your selection before continuing';
             } else if (message.includes('adding')) {
-              toastMsg =
-                "It seems like you don't have this network setup in MetaMask,";
               toastLink = {
-                title: 'please go here to add the network',
+                title:
+                  "It seems like you don't have this network setup in MetaMask, please go here to add the network",
```

Comment:
`"It seems like you don't have this network set up in MetaMask. Please go here to add the network."`

Small grammar change